.ci_scripts/beforeInstall.sh

```diff
@@ -30,21 +32,25 @@ fi
 
 echo "Max connection: $MAX_CON"
 
-# OpenSSL
```

Comment:
I prefer that to be reverted so the binary is the same in any case, even if the options are differents